### PR TITLE
feat: use local source for installing bonfire

### PR DESCRIPTION
**Pull request by Victoremepunto**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-  -o oc.tar.gz && \
-  tar -C /usr/bin/ -xvzf oc.tar.gz oc kubectl && \
-  rm -f oc.tar.gz
+USER 1001
+WORKDIR $APP_ROOT
 
-USER bonfire
-WORKDIR /opt/bonfire
-ENV PATH="/opt/bonfire/.local/bin:$PATH"
+COPY --chown=1001:0 . .
```
Comment (bsquizz) [inline Dockerfile:24]: This could potentially copy in the hidden `.` files right? Do you need to copy explicit paths here?

**Comment by adamrdrew** [comment]
Other than what Brandon pointed out this LGTM

File: Dockerfile (hunk 1 above)
Comment (Victoremepunto) [reply Dockerfile:24]: yeah actually due to how we handle versions, I do - bonfire runs this scm dynamic versioning, I need the git repo metadata to get the right version 

I was thinking building the wheel and installing that instead - but there's not much of a difference anyways

Comment (Victoremepunto) [reply Dockerfile:24]: that said, I still think a `.dockerignore` could really help keeping a bunch of not really needed files out, so I'll push that, thanks!

Comment (Victoremepunto) [reply Dockerfile]: I added a multi-stage build to build a wheel and then copy and install that alone, please can you give it a second look?

**Review by bsquizz — APPROVED** [review]
Looks good!

**Pull request merged**